draft-ietf-rtcweb-jsep.xml

```diff
@@ -3096,14 +3096,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         createOffer/createAnswer options or RtpTransceiver APIs MUST be
         used.</t>
 
-        <t>Note that the application MAY modify the SDP to reduce the
-        capabilities in the offer it sends to the far side
-        (post-setLocalDescription) or the offer that it installs from
-        the far side (pre-setRemoteDescription), as long as it remains
-        a valid SDP offer and specifies a subset of what was in the
-        original offer. This is safe because the answer is not
-        permitted to expand capabilities, and therefore will just
-        respond to what is present in the offer.</t>
+        <t>After calling setLocalDescription, the application MAY modify the SDP to
```

Comment:
Do we want to explicitly mention offers here, i.e.

> After calling setLocalDescription with an offer, ...

Or do we want to explicitly allow the same munging on answers (which almost certainly should be legal)?